tokio-trace-core/src/lib.rs

```diff
@@ -244,6 +176,10 @@ pub struct Event<'a> {
 /// [`Subscriber`]: ::Subscriber
 #[derive(Clone, Debug, Eq, Hash)]
 pub struct Meta<'a> {
+    // TODO: The fields on this type are currently `pub` because it must be able
+    // to be constructed statically by macros. However, when `const fn`s are
+    // available on stable Rust, this will no longer be necessary. Thus, these
+    // fields should be made private when `const fn` is stable.
```

Comment:
Making these fields private would be a breaking change, correct?

Reply:
Yes. In the context I think it's probably worth it.

Reply:
I agree, just wanted to ask about that.

Reply:
👍